components/brave_wallet/browser/asset_ratio_response_parser.cc

```diff
@@ -173,39 +171,6 @@ bool ParseAssetPriceHistory(const base::Value& json_value,
   return true;
 }
 
-mojom::BlockchainTokenPtr ParseTokenInfo(const base::Value& json_value,
-                                         const std::string& chain_id,
-                                         mojom::CoinType coin) {
-  auto token_info = api::asset_ratio::TokenInfo::FromValue(json_value);
```

Comment:
Let's also drop `TokenInfo` and `TokenInfoResult` from asset_ratio.idl

Reply:
ed535e0